docs/deploy/ALL_DOCS.md

```
@@ -0,0 +1,188 @@
+1. [SOLO.md](SOLO.md) (last-verified: 2025-11-02) (обзор)
+   ↓
+2. [GITHUB_SECRETS.md](GITHUB_SECRETS.md) (last-verified: 2025-11-02) (секреты)
+   ↓
+3. Деплой
+```
+
+---
+
+## 📂 Структура файлов в репозитории
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)